k8s/helm-chart/dynatrace-gcp-function/templates/active-gate-pod.yaml

```diff
@@ -0,0 +1,108 @@
+# Copyright 2021 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+{{- if or (eq .Values.deploymentType "logs") (eq .Values.deploymentType "all") }}
+  {{- if (.Values.activeGate.autoDeploy) }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: dynatrace-activegate-gcpmon-router
+  namespace: dynatrace
+spec:
+  type: ClusterIP
+  selector:
+    app: dynatrace-ag-gcpmon
```

Comment:
Good idea. I'll change the name of AG pod.  As it comes to the second one I'm not sure since by listing all resources u can strictly say what it is. For example **dynatrace-gcp-function-sa** or **dynatrace-gcp-function-secret**.

Reply:
okay, maybe we could replace the `router` with `service` to explicitly say what kind of Kubernetes resource it is?

Reply:
Changed